amethyst_controls/src/resources.rs

```diff
@@ -1,20 +1,16 @@
 use serde::{Deserialize, Serialize};
 
+use derive_new::new;
```

Comment:
Instead of using an `use` statement in each file, you can use the following in lib.rs:

```
#[macro_use]
extern crate derive_new;
```

This way, the macro will be globally exported for the amethyst_control crate :)